backend/core/src/shared/units-transformations.ts

```diff
@@ -244,7 +244,9 @@ const hmiData = (units: Units, maxHouseholdSize: number, amiCharts: AmiChart[])
 }
 
 const getCurrencyString = (initialValue: string) => {
-  return usd.format(getRoundedNumber(initialValue))
+  const roundedValue = getRoundedNumber(initialValue)
+  if (Number.isNaN(roundedValue)) return "N/A"
```

Comment:
If we return this string in English here, the translations won't come through. We may need to pass just the key (`"t.n/a"`) (like we are doing here: https://github.com/bloom-housing/bloom/blob/dev/ui-components/src/tables/StandardTable.tsx#L155) and translate on the frontend (which we allow here: https://github.com/bloom-housing/bloom/blob/dev/ui-components/src/tables/StandardTable.tsx#L155)